docker-compose.yaml

```diff
@@ -47,8 +47,42 @@ services:
                      -Ddatabase.dbname=mysql -Ddatabase.schema=openmrs -Ddatabase.serverId=77 -Ddatabase.offsetStorage=/tmp/offset.dat
                      -Ddatabase.databaseHistory=/tmp/dbhistory.dat -Dopenmrs.username=admin -Dopenmrs.password=Admin123
                      -Dopenmrs.serverUrl=http://localhost:8099 -Dopenmrs.fhirBaseEndpoint=/openmrs/ws/fhir2/R3
-                     -Dfhir.sinkPath=http://localhost:8098/fhir
+                     -Dfhir.sinkPath=http://localhost:8098/hapi-fhir-jpaserver/fhir
```

Comment:
As a side note, I think this won't work if you build/run from head because we switched to command line arguments. Updating the docker compose config was missed in that PR.